flow/scripts/prep_and_run_dream_place.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/tcsh
+cp ${SCRIPTS_DIR}/DREAMPlace_ref.json ${RESULTS_DIR}/DREAMPlace.json
+sed -i "s@xx_tech_lef_xx@${TECH_LEF}@" ${RESULTS_DIR}/DREAMPlace.json
+sed -i "s@xx_sc_lef_xx@${SC_LEF}@" ${RESULTS_DIR}/DREAMPlace.json
+cp ${RESULTS_DIR}/3_2_place_iop.def ${RESULTS_DIR}/2_DREAMPlace_input.def
+sed -i "/+ PORT/d" ${RESULTS_DIR}/2_DREAMPlace_input.def
+sed -i "s@xx_input_def_xx@${RESULTS_DIR}/2_DREAMPlace_input.def@" ${RESULTS_DIR}/DREAMPlace.json
+sed -i "s@xx_result_dir_xx@${RESULTS_DIR}@" ${RESULTS_DIR}/DREAMPlace.json
+
+sed -i "s@xx_place_density_xx@${PLACE_DENSITY}@" ${RESULTS_DIR}/DREAMPlace.json
+
+#if (! $?DREAM_PLACE_DENSITY ) then
+#    sed -i "/xx_place_density_xx/d" ${RESULTS_DIR}/DREAMPlace.json
+#else
+#    sed -i "s@xx_place_density_xx@${DREAM_PLACE_DENSITY}@" ${RESULTS_DIR}/DREAMPlace.json
+#endif
```

Comment:
Remove if not used

Reply:
Okay. 

Thank you, Vitor, for your feedback.